src/components/app/flyouts/settings/settings.js

```diff
@@ -170,7 +239,8 @@ export class Settings extends LinkedComponent {
                 !loading && hasChanged &&
                 <Btn type="submit" className="apply-button">{t('settingsFlyout.apply')}</Btn>
               }
-              <Btn svg={svgs.x} onClick={onClose} className="close-button">{hasChanged ? t('settingsFlyout.cancel') : t('settingsFlyout.close')}</Btn>
+              <Btn svg={svgs.x} onClick={onClose} className="close-button">
+                {hasChanged ? t('settingsFlyout.cancel') : t('settingsFlyout.close')}</Btn>
```

Comment:
What changed here?

Reply:
Coding Standard: Max 90 to 110 chars per line